config/default/manager_credentials_patch.yaml

```diff
@@ -9,12 +9,23 @@ spec:
       containers:
       - name: manager
         env:
-        - name: GOOGLE_APPLICATION_CREDENTIALS
-          value: /home/.azure/credentials.json
-        volumeMounts:
-        - name: credentials
-          mountPath: /home/.azure
-      volumes:
-      - name: credentials
-        secret:
-          secretName: manager-bootstrap-credentials
+          - name: AZURE_SUBSCRIPTION_ID
```

Comment:
@justaugustus @vincepri does this make more sense?

Reply:
I think so, +1